2024-03-24 14:16:36,077 INFO    MainThread:14912 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2024-03-24 14:16:36,077 INFO    MainThread:14912 [wandb_setup.py:_flush():76] Configure stats pid to 14912
2024-03-24 14:16:36,078 INFO    MainThread:14912 [wandb_setup.py:_flush():76] Loading settings from C:\Users\raul\.config\wandb\settings
2024-03-24 14:16:36,078 INFO    MainThread:14912 [wandb_setup.py:_flush():76] Loading settings from C:\Users\raul\Universidad Pontificia Comillas\Javier Álvarez Martínez - MachineLoooorning\wandb\settings
2024-03-24 14:16:36,078 INFO    MainThread:14912 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-24 14:16:36,078 INFO    MainThread:14912 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-24 14:16:36,081 INFO    MainThread:14912 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'DeepLearning\\mainTraining.py', 'program_abspath': 'C:\\Users\\raul\\Universidad Pontificia Comillas\\Javier Álvarez Martínez - MachineLoooorning\\DeepLearning\\mainTraining.py', 'program': 'C:\\Users\\raul\\Universidad Pontificia Comillas\\Javier Álvarez Martínez - MachineLoooorning\\DeepLearning\\mainTraining.py'}
2024-03-24 14:16:36,082 INFO    MainThread:14912 [wandb_init.py:_log_setup():526] Logging user logs to C:\Users\raul\Universidad Pontificia Comillas\Javier Álvarez Martínez - MachineLoooorning\wandb\run-20240324_141636-c11kchmk\logs\debug.log
2024-03-24 14:16:36,082 INFO    MainThread:14912 [wandb_init.py:_log_setup():527] Logging internal logs to C:\Users\raul\Universidad Pontificia Comillas\Javier Álvarez Martínez - MachineLoooorning\wandb\run-20240324_141636-c11kchmk\logs\debug-internal.log
2024-03-24 14:16:36,083 INFO    MainThread:14912 [wandb_init.py:init():566] calling init triggers
2024-03-24 14:16:36,083 INFO    MainThread:14912 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-24 14:16:36,083 INFO    MainThread:14912 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 232py1ls
2024-03-24 14:16:36,084 INFO    MainThread:14912 [wandb_run.py:_finish():1970] finishing run raulgonzalez/ImagesClassification/232py1ls
2024-03-24 14:16:36,085 INFO    MainThread:14912 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-24 14:16:36,085 INFO    MainThread:14912 [wandb_run.py:_restore():2193] restore
2024-03-24 14:16:36,086 INFO    MainThread:14912 [wandb_run.py:_restore():2199] restore done
2024-03-24 14:16:44,777 INFO    MainThread:14912 [wandb_run.py:_footer_history_summary_info():3851] rendering history
2024-03-24 14:16:44,777 INFO    MainThread:14912 [wandb_run.py:_footer_history_summary_info():3883] rendering summary
2024-03-24 14:16:44,789 INFO    MainThread:14912 [wandb_run.py:_footer_sync_info():3810] logging synced files
2024-03-24 14:16:44,828 INFO    MainThread:14912 [wandb_init.py:init():616] starting backend
2024-03-24 14:16:44,828 INFO    MainThread:14912 [wandb_init.py:init():620] setting up manager
2024-03-24 14:16:44,844 INFO    MainThread:14912 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-24 14:16:44,845 INFO    MainThread:14912 [wandb_init.py:init():628] backend started and connected
2024-03-24 14:16:44,872 INFO    MainThread:14912 [wandb_init.py:init():720] updated telemetry
2024-03-24 14:16:44,877 INFO    MainThread:14912 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-24 14:16:47,630 INFO    MainThread:14912 [wandb_run.py:_on_init():2262] communicating current version
2024-03-24 14:16:49,183 WARNING MainThread:14912 [wandb_init.py:init():1198] interrupted
Traceback (most recent call last):
  File "C:\Users\raul\.conda\envs\noestructurados\Lib\site-packages\wandb\sdk\wandb_init.py", line 1176, in init
    run = wi.init()
          ^^^^^^^^^
  File "C:\Users\raul\.conda\envs\noestructurados\Lib\site-packages\wandb\sdk\wandb_init.py", line 802, in init
    run._on_init()
  File "C:\Users\raul\.conda\envs\noestructurados\Lib\site-packages\wandb\sdk\wandb_run.py", line 2266, in _on_init
    version_result = version_handle.wait(timeout=30)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\raul\.conda\envs\noestructurados\Lib\site-packages\wandb\sdk\lib\mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\raul\.conda\envs\noestructurados\Lib\site-packages\wandb\sdk\lib\mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\raul\.conda\envs\noestructurados\Lib\site-packages\wandb\sdk\lib\mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\raul\.conda\envs\noestructurados\Lib\threading.py", line 629, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\raul\.conda\envs\noestructurados\Lib\threading.py", line 331, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-03-24 14:16:54,878 WARNING MsgRouterThr:14912 [router.py:message_loop():77] message_loop has been closed
